components/Prose.vue

```diff
@@ -6,7 +6,6 @@
       prose-h2:scroll-mt-16 prose-h2:mt-8 prose-h2:pt-8 prose-h2:border-t
       prose-h3:scroll-mt-20
       prose-lead:text-slate-500
-      prose-a:no-underline
```

Comment:
There is a regression here with our markdown headings which are also links:
<img width="996" alt="Screenshot 2023-01-18 at 9 21 24 AM" src="https://user-images.githubusercontent.com/64108933/213004591-8c8055e9-eef6-4c28-9248-41e66eab4349.png">